generators/client/templates/angular/src/main/webapp/_manifest.webapp

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "<%=angular2AppName%>",
+  "short_name": "<%=angular2AppName%>",
+  "icons": [
+        {
+            "src": "/content/images/logo-jhipster.png",
+            "sizes": "192x192",
+            "type": "image/png",
+            "density": 4
```

Comment:
yeah I know that part. But I don't want to overcomplicate things here. But yes I do agree we can have those 5 combinations which satisfies all the needs.